src/glum/_glm.py

```diff
@@ -1328,6 +1334,401 @@ def predict(
         sample_weight = _check_weights(sample_weight, X.shape[0], X.dtype)
         return mu * sample_weight
 
+    def coef_table(
+        self,
+        significance_level=0.05,
+        X=None,
+        y=None,
+        mu=None,
+        offset=None,
+        sample_weight=None,
+        dispersion=None,
+        robust=None,
+        clusters: np.ndarray = None,
+        expected_information=None,
+    ):
+        """Get a table of of the regression coefficients.
+
+        Includes coefficient estimates, standard errors, t-values, p-values
+        and confidence intervals.
+
+        Parameters
+        ----------
+        significance_level : float, optional, default=0.05
```

Comment:
What about instead having an argument `confidence_level` with default value 0.95? This seems to be slightly more common language (e.g., the [Wikipedia article](https://en.wikipedia.org/wiki/Confidence_interval) only mentions confidence levels) "Significance level" is used mostly in the context of hypothesis testing.